governance/bylaws.md

```diff
@@ -186,11 +189,46 @@ The time and place of every annual or special meeting shall be given to each mem
 A quorum for the transaction of business at any annual or special meeting of members of the Association shall be fifteen (15) Regular or Certified Members in good standing including at least one half of the members of the Board.  If a quorum is not present, the meeting shall be automatically adjourned and the board must convene another meeting within forty five (45) days.
 
 #### 3.4	Voting
-
 All resolutions or motions at an annual or special meeting of members of the Association regarding dissolution, amendments to the Bylaws, or changes to the Code of Ethics shall be approved by a two-thirds majority of the votes cast by Regular or Fellow Members.
 All resolutions or motions at an annual or special meeting of members of the Association regarding dissolution, amendments to the Common Body of Knowledge and Certification/Re-certification Standards shall be approved by a two-thirds majority of the votes cast by Certified Members.
 All other resolutions or motions at an annual or special meeting of members of the Association shall be passed or rejected by a simple majority of the votes cast by Regular or Fellow Members.
-At the discretion of the Board, votes may be cast in person at the meeting, or received by the Secretary by mail or electronic transmission twenty-four (24) hours in advance of the meeting unless otherwise required by the Bylaws of the Association. 
+Votes may be cast by,
+1. indicating Yea/Nay, either in person, by proxy, telephonically or electronically
```

Comment:
I added this to include people attending via phone who, obviously, can't be counted if it's just a hand raise.